src/error/GraphQLError.d.ts

```diff
@@ -4,6 +4,9 @@ import { ASTNode } from '../language/ast';
 import { Source } from '../language/source';
 import { SourceLocation } from '../language/location';
 
+// eslint-disable-next-line import/no-cycle
+import { GraphQLFormattedError } from './formatError';
```

Comment:
Might be better to move the definition of GraphQLFormattedError here to avoid the cycle; then you can re-export it from `formatError` for backwards compatibility.

Reply:
I'll have to move both formatError and the type, sure - give me a sec.